mmv1/third_party/terraform/services/compute/data_source_google_compute_subnetworks_test.go

```diff
@@ -11,6 +11,157 @@ import (
 	"github.com/hashicorp/terraform-provider-google/google/envvar"
 )
 
+func TestAccDataSourceGoogleSubnetworks_update(t *testing.T) {
```

Comment:
I'm sorry for not catching this before; this test is currently among the tests for the `google_compute_subnetworks` pural [_data source_](https://developer.hashicorp.com/terraform/language/data-sources) but it should be in the tests for the `google_compute_subnetwork` [_resource_](https://developer.hashicorp.com/terraform/language/resources), which live in https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/services/compute/resource_compute_subnetwork_test.go.tmpl.

You could add updating the purpose field to the existing TestAccComputeSubnetwork_update test instead of adding a new test.